docs/source/amp.rst

```diff
@@ -256,6 +256,100 @@ In this case, combine the two layers using :func:`torch.nn.functional.binary_cro
 or :mod:`torch.nn.BCEWithLogitsLoss`.  ``binary_cross_entropy_with_logits`` and ``BCEWithLogits``
 are safe to autocast.
 
+.. _autocast-xpu-op-reference:
+
+XPU Op-Specific Behavior
+------------------------
+The following lists describe the behavior of eligible ops in autocast-enabled regions.
+These ops always go through autocasting whether they are invoked as part of a :class:`torch.nn.Module`,
+as a function, or as a :class:`torch.Tensor` method. If functions are exposed in multiple namespaces,
+they go through autocasting regardless of the namespace.
+
+Ops not listed below do not go through autocasting.  They run in the type
+defined by their inputs.  However, autocasting may still change the type
+in which unlisted ops run if they're downstream from autocasted ops.
+
+If an op is unlisted, we assume it's numerically stable in ``float16``.
+If you believe an unlisted op is numerically unstable in ``float16``,
+please file an issue.
+
+XPU Ops that can autocast to ``float16``
+""""""""""""""""""""""""""""""""""""""""
+
+``addbmm``,
```

Comment:
I do not think that some of these operations are currently actually supported by XPU. I don't see neither in explicit fallback list nor in a list of implemented functions. And associated tests are skipped. **Thus, I think this PR might be documenting something which is currently not supported which is bad and will confuse end users.** @jingxu10 can you, please, discuss supported ops with the dev team?

For example:
```
$ fgrep -rsn addbmm
test/xpu/run_test_with_skip.py:426:    "test_dtypes_addbmm_xpu",
test/xpu/run_test_with_skip.py:431:    "test_dtypes_baddbmm_xpu",
test/xpu/run_test_with_skip.py:483:    "test_noncontiguous_samples_addbmm_xpu_complex64",
test/xpu/run_test_with_skip.py:484:    "test_noncontiguous_samples_addbmm_xpu_float32",
test/xpu/run_test_with_skip.py:485:    "test_noncontiguous_samples_addbmm_xpu_int64",
test/xpu/run_test_with_skip.py:495:    "test_noncontiguous_samples_baddbmm_xpu_complex64",
test/xpu/run_test_with_skip.py:496:    "test_noncontiguous_samples_baddbmm_xpu_int64",
test/xpu/run_test_with_skip.py:567:    "test_numpy_ref_addbmm_xpu_complex128",
test/xpu/run_test_with_skip.py:568:    "test_numpy_ref_addbmm_xpu_float64",
test/xpu/run_test_with_skip.py:569:    "test_numpy_ref_addbmm_xpu_int64",
test/xpu/run_test_with_skip.py:571:    "test_out_addbmm_xpu_float32",
test/xpu/run_test_with_skip.py:574:    "test_out_baddbmm_xpu_float32",
test/xpu/run_test_with_skip.py:577:    "test_out_requires_grad_error_addbmm_xpu_complex64",
test/xpu/run_test_with_skip.py:581:    "test_out_requires_grad_error_baddbmm_xpu_complex64",
test/xpu/run_test_with_skip.py:617:    "test_out_warning_baddbmm_xpu",
test/xpu/run_test_with_skip.py:645:    "test_variant_consistency_eager_baddbmm_xpu_complex64",
test/xpu/run_test_with_skip.py:646:    "test_variant_consistency_eager_baddbmm_xpu_float32",
test/xpu/run_test_with_skip.py:710:    "test_conj_view_addbmm_xpu_complex64",
test/xpu/run_test_with_skip.py:711:    "test_neg_conj_view_addbmm_xpu_complex128",
test/xpu/run_test_with_skip.py:732:    "test_conj_view_baddbmm_xpu_complex64",
test/xpu/run_test_with_skip.py:798:    "test_neg_conj_view_baddbmm_xpu_complex128",
test/xpu/run_test_with_skip.py:852:    "test_neg_view_addbmm_xpu_float64",
test/xpu/run_test_with_skip.py:857:    "test_neg_view_baddbmm_xpu_float64",
```

cc: @EikanWang , @jgong5, @fengyuan14

Reply:
@dvrogozh , sorry for the late reply. Regarding mm-related operators for XPU, we have upstreamed to the stock PyTorch

Reply:
@dvrogozh any further comments? Can we merge the PR?

Reply:
@EikanWang : mm operations are a subset of operations in this list. I am trying to raise a broader question: are all operations listed here implemented to run on XPU device as of 97d692eb8c4b3afab17700a2fd918adcea0cba45 which is current commit in https://github.com/pytorch/pytorch/blob/main/third_party/xpu.txt? I do not see some of them in the https://github.com/intel/torch-xpu-ops/blob/main/yaml/xpu_functions.yaml, but I might not be fully aware of the  design and, hence miss them. If you believe are all are implemented, I am fine with this PR. Otherwise I believe this list should be sanitized to only those ops which are actually supported to run on XPU.

Reply:
for matmul/conv, https://github.com/pytorch/pytorch/tree/6206da55ef1e2477eea9b0467c1f68d21cad65c0/aten/src/ATen/native/mkldnn/xpu FYI

Reply:
> for matmul/conv

Thank you.